spec/authoring_spec.ts

```diff
@@ -716,6 +724,62 @@ describe('authoring', () => {
         expect(authoring.getAbstractText()).toBe('item6 text');
     });
 
+    it('Not modifying crops will not trigger an article change', () => {
```

Comment:
Good job on writing tests decoupled from the DOM structure. Any feedback on writing them this way with these helpers? Do you have suggestions for improvement?

Reply:
No suggestions, it was a good experience overall, also they make tests more readable